packages/react/src/utils/assert.ts

```diff
@@ -4,9 +4,11 @@ export function assert(condition: boolean, message: string): asserts condition {
   }
 }
 
-export const assertMessageUseErrorBoundaryOnlyInChildrenOfErrorBoundary =
-  'useErrorBoundary: this hook should be called in ErrorBoundary.props.children'
-export const assertMessageUseErrorBoundaryFallbackPropsOnlyInFallbackOfErrorBoundary =
-  'useErrorBoundary: this hook should be called in ErrorBoundary.props.children'
-export const assertMessageUseErrorBoundaryGroupOnlyInChildrenOfErrorBoundaryGroup =
+export const useErrorBoundary_this_hook_should_be_called_in_ErrorBoundary_props_children =
   'useErrorBoundary: this hook should be called in ErrorBoundary.props.children'
+
+export const useErrorBoundaryFallbackProps_this_hook_should_be_called_in_ErrorBoundary_props_fallback =
+  'useErrorBoundaryFallbackProps: this hook should be called in ErrorBoundary.props.fallback'
+
+export const useErrorBoundaryGroup_this_hook_should_be_called_in_ErrorBoundary_props_children =
+  'useErrorBoundaryGroup: this hook should be called in ErrorBoundary.props.children'
```

Comment:
Oh,, my mistake, it's assert message for useErrorBoundaryFallbackProps, useErrorBoundaryGroup. This will fix them